src/util/SearchUtils.ts

```diff
@@ -38,7 +45,43 @@ export function isValidInput(keyword: string) {
     }
 }
 
-export function normalizeInputFormat(keyword: string) {
+export function isSearchingByRsid(keyword: string) {
+    // input should start with "rs"
+    return keyword.startsWith('rs');
+}
+
+export function isSearchingByHgvsg(keyword: string) {
+    // input should contain ":g."
+    return /:g./i.test(keyword);
+}
+
+export function normalizeInputFormatForInternalDatabaseSearch(keyword: string) {
```

Comment:
Would be nice to have some unit tests for these kind of complex utility functions.

Reply:
Currently we don't have any unit tests, but good to add some soon https://github.com/genome-nexus/genome-nexus/issues/573